client/app/components/Form/__tests__/DynamicForm.spec.jsx

```diff
@@ -7,7 +7,7 @@ import { InputMocks } from '../../../mocks/InputMocks';
 
 // TODO (julianguyen): Include InputTextarea after writing stubs for pell editor
 
-const mockInputs = [
+const defualtMockInputs = [
```

Comment:
Looks like there's a typo, it should be `defaultMockInputs` instead.

Reply:
Thanks for the review. fixed in the latest commit.